gitops/git/github/github.go

```diff
@@ -44,14 +51,24 @@ func CreatePR(from, to, title string) error {
 		MaintainerCanModify: new(bool),
 		Draft:               new(bool),
 	}
-	_, resp, err := gh.PullRequests.Create(ctx, *repoOwner, *repo, pr)
-	defer resp.Body.Close()
-	body, err := ioutil.ReadAll(resp.Body)
-	log.Print("github response: ", string(body))
```

Comment:
If it was a newly created PR, the `resp.Body` is already consumed and closed in creating the PR object.

For example, we would get the following error message.
```
2021/04/11 10:32:12 create_gitops_prs.go:228: unable to create PR: http: read on closed response body
```

Reply:
Interesting - I don't see this same read on closed response body error for new PRs (just an empty response body).  The reorganization of the error checking here makes things clearer regardless though 👍

Reply:
probably a different github version. The github version for the enterprise is usually a few versions behind the public one.